Traceback (most recent call last):
  File "/home/vagrant/.local/lib/python3.10/site-packages/selenium/webdriver/common/driver_finder.py", line 67, in _binary_paths
    output = SeleniumManager().binary_paths(self._to_args())
  File "/home/vagrant/.local/lib/python3.10/site-packages/selenium/webdriver/common/selenium_manager.py", line 53, in binary_paths
    return self._run(args)
  File "/home/vagrant/.local/lib/python3.10/site-packages/selenium/webdriver/common/selenium_manager.py", line 118, in _run
    raise WebDriverException(
selenium.common.exceptions.WebDriverException: Message: Unsuccessful command executed: /home/vagrant/.local/lib/python3.10/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --language-binding python --output json; code: 65
{'code': 65, 'message': 'error sending request for url (https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json)', 'driver_path': '', 'browser_path': ''}



The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/vagrant/shared/traffic_generator_guest.py", line 356, in <module>
    driver = webdriver.Chrome()
  File "/home/vagrant/.local/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py", line 45, in __init__
    super().__init__(
  File "/home/vagrant/.local/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py", line 50, in __init__
    if finder.get_browser_path():
  File "/home/vagrant/.local/lib/python3.10/site-packages/selenium/webdriver/common/driver_finder.py", line 47, in get_browser_path
    return self._binary_paths()["browser_path"]
  File "/home/vagrant/.local/lib/python3.10/site-packages/selenium/webdriver/common/driver_finder.py", line 78, in _binary_paths
    raise NoSuchDriverException(msg) from err
selenium.common.exceptions.NoSuchDriverException: Message: Unable to obtain driver for chrome; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location

